`npm start` 失败

git clone 到本地后，npm start 报错：

```
C:\Users\Jeff\react-codebox>npm start

> @axetroy/react-codebox@0.1.0 start C:\Users\Jeff\react-codebox
> webpack-dev-server --config webpack.config.dev.js

C:\Users\Jeff\react-codebox\node_modules\webpack-cli\bin\config-yargs.js:89
                                describe: optionsSchema.definitions.output.properties.path.description,
                                                                           ^

TypeError: Cannot read property 'properties' of undefined
    at module.exports (C:\Users\Jeff\react-codebox\node_modules\webpack-cli\bin\config-yargs.js:89:48)
    at Object.<anonymous> (C:\Users\Jeff\react-codebox\node_modules\webpack-dev-server\bin\webpack-dev-server.js:65:25)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:643:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:839:10)
    at internal/main/run_main_module.js:17:11
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @axetroy/react-codebox@0.1.0 start: `webpack-dev-server --config webpack.config.dev.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @axetroy/react-codebox@0.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Jeff\AppData\Roaming\npm-cache\_logs\2020-02-18T07_11_49_418Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -33,7 +33,7 @@
     "rimraf": "^2.6.2",
     "style-loader": "^0.19.1",
     "webpack": "^4.6.0",
-    "webpack-cli": "^2.0.15",
+    "webpack-cli": "^3.3.11",
     "webpack-dev-server": "^3.1.3"
   }
 }

